src/internal/SwitchBase.js

```diff
@@ -155,6 +155,10 @@ export default function createSwitch(
     }
   }
 
+   /**
+   ** NB: If changed, please update Checkbox, Switch and Radio
+   ** so that the API documentation is updated.
+   **/
```

Comment:
This component wasn't done by me, should I edit it?
It's causing more lint errors.